File: mobile/src/main/java/com/kyleriedemann/hiphopairhorn/AirHornApplication.java
package com.kyleriedemann.hiphopairhorn;

import android.app.Application;

import com.crashlytics.android.Crashlytics;
import com.crashlytics.android.answers.Answers;

import io.fabric.sdk.android.Fabric;
import timber.log.Timber;

public class AirHornApplication extends Application {
    @Override
    public void onCreate() {
        super.onCreate();
        Fabric.with(this, new Crashlytics());
        Fabric.with(this, new Answers());

        Timber.plant(new Timber.DebugTree());
    }
}

File: mobile/src/main/java/com/kyleriedemann/hiphopairhorn/AirHornApplication.kt
package com.kyleriedemann.hiphopairhorn

import android.annotation.SuppressLint
import android.app.Application
import android.util.Log

import timber.log.Timber

class AirHornApplication : Application() {
    @SuppressLint("LogNotTimber")
    override fun onCreate() {
        super.onCreate()
        Timber.plant(Timber.DebugTree())
        Timber.i("Planted debug tree")
        Log.d("Timber", "Should have planted debug tree")
    }
}
